deps/wazuh_testing/wazuh_testing/qa_docs/lib/config.py

```diff
@@ -29,21 +33,44 @@ def __init__(self, config_file):
         self.test_cases_field = None
 
         try:
-            with open(config_file) as fd:
+            with open(args[0]) as fd:
                 self._config_data = yaml.safe_load(fd)
         except:
             logging.error("Cannot load config file")
             raise Exception("Cannot load config file")
 
-        self._read_project_path()
+        self._read_function_regex()
+        self._read_output_fields()
+        self._read_test_cases_field()
         self._read_documentation_path()
         self._read_include_paths()
         self._read_include_regex()
         self._read_group_files()
-        self._read_function_regex()
         self._read_ignore_paths()
-        self._read_output_fields()
-        self._read_test_cases_field()
+
+        if len(args) >= 3:
+            self.documentation_path = args[2]
+        if len(args) == 4:
+            # It is called with a single test to parse
+            self.mode = mode.SINGLE_TEST
+            self.test_name = args[3]
+            self._read_test_info()
+            self._read_module_info()
+
+
+    def _read_test_info(self):
+        '''
+        brief: Reads from the config file the keys to be printed from test info
+        '''
+        if 'Test info' in self._config_data:
+            self.test_info = self._config_data['Test info']
+    
+    def _read_module_info(self):
+        '''
+        brief: Reads from the config file the keys to be printed from module info
+        '''
+        if 'Module info' in self._config_data:
```

Comment:
Never use this type of variable. Always use lowercase and no blanks. Use `module_info` not `Module info`